devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -174,6 +183,9 @@ private addChildButtons(numberOfButtons) {
 		try {
 			String childDni = "${device.deviceNetworkId}:$endpoint"
 			def componentLabel = (device.displayName.endsWith(' 1') ? device.displayName[0..-2] : device.displayName) + "${endpoint}"
+			if (isAduroSmartRemote()) {
+				componentLabel = device.displayName + " child button ${endpoint}"
```

Comment:
just so you know, your display names will be absurdly long and most likely be truncated, making it difficult to determine which button is which. For example:

`Eria scene button switch V2.0 child button 1`

Perhaps consider changing your join names to `Eria Remote` or something like that. If you like it how it is, that's fine, I'm just warning you about usability concerns I have.

Reply:
i using " - " instead of "child button"